src/backend/gl/src/queue.rs

```diff
@@ -521,28 +521,20 @@ impl CommandQueue {
                     .clear_buffer_i32_slice(glow::COLOR, draw_buffer, &mut cv);
             },
             com::Command::ClearBufferDepthStencil(depth, stencil) => unsafe {
+                let gl = &self.share.context;
                 match (depth, stencil) {
                     (Some(depth), Some(stencil)) => {
-                        self.share.context.clear_buffer_depth_stencil(
-                            glow::DEPTH_STENCIL,
-                            0,
-                            depth,
-                            stencil as _,
-                        );
+                        gl.depth_mask(true);
```

Comment:
are you sure we don't need to reset this back? gfx-rs API has 3 kind of clears, just like Vulkan:
  - explicit command outside of a render pass - you don't need to restore the mask in this case
  - load operation on an attachment when starting a render pass - you may need to restore the mask according to the actively bound pipeline
  - explicit attachment clear command inside a render pass - you do need to restore the mask

Reply:
I don't have a good understandting of the whole API yet so thanks for pointing this out

Reply:
if you can hop on #gfx:matrix.org, we'll be happy to explain in a more interactive way :)